services/web/server/src/simcore_service_webserver/projects/_controller/nodes_rest.py

```diff
@@ -323,18 +329,41 @@ async def _stop_dynamic_service_task(
         return web.json_response(status=status.HTTP_204_NO_CONTENT)
 
     except (RPCServerError, ServiceWaitingForManualInterventionError) as exc:
-        # in case there is an error reply as not found
-        raise web.HTTPNotFound(text=f"{exc}") from exc
+        error_code = getattr(exc, "error_code", None) or create_error_code(exc)
+        user_error_msg = user_message(
+            f"Could not stop dynamic service {dynamic_service_stop.project_id}.{dynamic_service_stop.node_id}"
+        )
+        _logger.exception(
+            **create_troubleshootting_log_kwargs(
+                user_error_msg,
+                error=exc,
+                error_code=error_code,
+                error_context={
+                    "project_id": dynamic_service_stop.project_id,
+                    "node_id": dynamic_service_stop.node_id,
+                    "user_id": dynamic_service_stop.user_id,
+                    "save_state": dynamic_service_stop.save_state,
+                    "simcore_user_agent": dynamic_service_stop.simcore_user_agent,
+                },
+            )
+        )
+        # ANE: in case there is an error reply as not found
+        return create_error_response(
+            error=ErrorGet(
+                message=user_error_msg,
+                support_id=IDStr(error_code),
+                status=status.HTTP_404_NOT_FOUND,
+            ),
+            status_code=status.HTTP_404_NOT_FOUND,
+        )
 
     except ServiceWasNotFoundError:
         # in case the service is not found reply as all OK
         return web.json_response(status=status.HTTP_204_NO_CONTENT)
 
 
 def register_stop_dynamic_service_task(app: web.Application) -> None:
-    TaskRegistry.register(
-        _stop_dynamic_service_task, allowed_errors=(web.HTTPNotFound,), app=app
-    )
+    TaskRegistry.register(_stop_dynamic_service_task, app=app)
```

Comment:
here I am just wondering about this.
AFAIK this is a long running task via RabbitMQ. and the `allowed_errors` are a tuple of error types that can go through the RabbitMQ and are interpreted on the lrt client side. please double check with @GitHK

Reply:
thx. Yes, I am aware of that. I also found why @GitHK  is using this. I asked him to talk offline to double check my proposed solution